ref-filter.c

```diff
@@ -1637,6 +1651,12 @@ void show_ref_array_item(struct ref_array_item *info, const char *format, int qu
 	putchar('\n');
 }
 
+/* Wrapper: Show ref_array_item w/o referencing container in function name */
```

Comment:
Let's just see if they complain. I couldn't find anything related in the CodingGuidelines.
They do mention: ["When you come up with an API, document it."](https://github.com/git/git/blob/master/Documentation/CodingGuidelines#L334)